src/matric/ck.py

```diff
@@ -0,0 +1,23 @@
+from src.utils import Subprocess
+from src.utils import copy_files
+from .index import MetricConfig
+
+
+class CK:
+    """
+    example:
+    java -jar ck-0.6.3-SNAPSHOT-jar-with-dependencies.jar /Users/ahmedalasifer/Desktop/FIT4003/scala
```

Comment:
What does `/Users/ahmedalasifer/Desktop/FIT4003/scala` mean in this setting? The output dir?

Reply:
The path in which the code to be analysed is located.

Reply:
the output in CK is stored under src, then i have to move it using mv

Reply:
but when we will be running SourceMeter, we can specify the output location

Reply:
Btw, what do I have to install to get CK working? Could you add some info into README.md on setup guide.